api/src/main/java/org/openmrs/FieldType.java

```diff
@@ -9,17 +9,35 @@
  */
 package org.openmrs;
 
+import javax.persistence.AttributeOverride;
+import javax.persistence.AttributeOverrides;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
 /**
  * FieldType
  */
+@Entity
+@Table(name = "field_type")
+@AttributeOverrides(value = {
+		@AttributeOverride(name = "name", column = @Column(name = "name", length = 50, nullable = false)),
+		@AttributeOverride(name = "retired", column = @Column(name = "retired", columnDefinition = "boolean default false"))
+})
 public class FieldType extends BaseChangeableOpenmrsMetadata {
 	
 	public static final long serialVersionUID = 35467L;
 	
 	// Fields
-	
+	@Id
+	@Column(name = "field_type_id")
```

Comment:
do all your attributes correspond with what he have in the hibernate mapping file that was removed?

Reply:
No @HerbertYiga dosent seem so from the build errors.